requests/models.py

```diff
@@ -335,17 +337,11 @@ def prepare_method(self, method):
         self.method = method
         if self.method is None:
             raise ValueError('Request method cannot be "None"')
-        self.method = to_native_string(self.method).upper()
+        self.method = to_native_string(self.method.upper())
```

Comment:
Looks like parens may have been unintentionally moved here? I don't know if it's actually going to affect the final output but may cause issues in future merges.